### DISCUSSION TITLE
Is it possible to replace page editor with the custom one?

### DISCUSSION THREAD

**Discussion by sentance** [swellstores/community#509] [Help] [answered]
Hello, I'm interested in replacing the default editor for the product pages with our own. Is that possible?

**Comment by logeshswell** [comment]
Do you mean the storefront theme editor?

**Reply by logeshswell** [reply, marked as answer]
Anyways, you can't replace your custom editor in the Swell admin dashboard. But, you can have your custom editor separately and make updates using Swell backend APIs.
